app/src/main/res/values/strings.xml

```diff
@@ -213,6 +213,7 @@
   <string name="admin_pin_error_pin_length">Your PIN should be 5 digits long.</string>
   <string name="admin_pin_error_pin_confirm_wrong">Please make sure that both PINs match.</string>
   <string name="save">Save</string>
+  <string name="admin_pin_activity_title">Authorise to add profiles</string>
```

Comment:
@rt4914 Shouldn't this be added in the lexicographical order?

Reply:
Currently we are not focused on that as it is hard to maintain without some form of CI or lint check.

Reply:
Okay, sounds good to me.
@starboi02 You can resolve this conversation.

Reply:
@rt4914 @prayutsu thanks !